crates/findex/Cargo.toml

```diff
@@ -15,7 +15,7 @@ name = "cloudproof_findex"
 [features]
 cloud = ["serde", "reqwest"]
 ffi = ["serde", "serde_json", "cloud", "cosmian_ffi", "tokio"]
-python = ["pyo3"]
+python = ["pyo3", "cloud", "tokio"]
```

Comment:
Do you want to ship "cloud" all the time?

Reply:
I think it's more convenient this way.
Plus it's the same as the ffi and wasm.